modules/bitgo/example/ts/eth/list-wallet-addresses.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * List deployed receive addresses of a eth wallet and their given token balances.
+ *
+ * This tool will help you see how to use the BitGo API to easily list your
+ * BitGo wallets.
+ *
+ * Copyright 2022, BitGo, Inc.  All Rights Reserved.
+ */
+import { BitGo } from 'bitgo';
+const bitgo = new BitGo({ env: 'test' });
+
+const coin = 'teth';
```

Comment:
same comment as in js file for this example

Reply:
Done.